TF_Service_dotNet/TouchFree_Service/Connection/ClientConnection.cs

```diff
@@ -269,38 +272,45 @@ protected void ProcessHandshake(ActionCode action, string requestContent)
             string clientApiVersion = (string)contentObj[VersionInfo.API_HEADER_NAME];
             Compatibility compatibility = GetVersionCompability(clientApiVersion, VersionInfo.ApiVersion);
 
+            string configurationWarning = string.Empty;
+
+            if (!configManager.AreConfigsInGoodState())
+            {
+                configurationWarning = " Configuration is in a bad state. Please update the settings via the Settings UI";
+            }
+
             switch (compatibility)
             {
                 case Compatibility.COMPATIBLE:
                     HandshakeCompleted = true;
                     response.status = "Success";
-                    response.message = "Handshake Successful";
+                    response.message = "Handshake Successful." + configurationWarning;
```

Comment:
Should each of these response messages include a space between the string and the warning?

e.g. "Handshake Successful. " + configurationWarning;

Reply:
I've included the space where the warning is set so that it only has the space when there is a warning